subsystems/drivebase/SwerveDrive.java

```diff
@@ -254,23 +296,23 @@ public void setModuleStates(SwerveModuleState[] states) {
         // these
         // values.
 
-        if (Math.abs(frontLeftSpeed) <= 0.01 && Math.abs(frontRightSpeed) <= 0.01
-        && Math.abs(backLeftSpeed) <= 0.01 && Math.abs(backRightSpeed) <= 0.01) {
-        frontLeftAngle = frontLeftPrevAngle;
-        frontRightAngle = frontRightPrevAngle;
-        backLeftAngle = backLeftPrevAngle;
-        backRightAngle = backRightPrevAngle;
-        }
+        // if (Math.abs(frontLeftSpeed) <= 0.01 && Math.abs(frontRightSpeed) <= 0.01
+        // && Math.abs(backLeftSpeed) <= 0.01 && Math.abs(backRightSpeed) <= 0.01) {
+        // frontLeftAngle = frontLeftPrevAngle;
+        // frontRightAngle = frontRightPrevAngle;
```

Comment:
why is this being commented out in this change? If we actually want to get rid of the code, just get rid of the code. But this is another change that doesn't seem related to advantagekit and I don't understand why it's here